### PR TITLE
Rename to Microsoft.VisualStudio.Web

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @prafullbhosale

**Comment by pranavkm** [comment]
@Eilon - there were a bunch of places in code with the Scaffolding package names \ assembly names were used as strings. It seemed easier to do a bulk update of the names rather than hunt these down and risk missing some.

**Comment by Eilon** [comment]
Understood.

And :shipit: (I reviewed all the changes using CodeFlow).

File: src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/Constants.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ internal static class Constants
         public const string CodeFileExtension = ".cs";
         public const string RazorTemplateExtension = ".cshtml";
 
-        public const string ThisAssemblyName = "Microsoft.Extensions.CodeGenerators.Mvc";
+        public const string ThisAssemblyName = "Microsoft.VisualStudio.Web.CodeGenerators";
```
Comment (prafullbhosale) [inline src/Microsoft.VisualStudio.Web.CodeGenerators.Mvc/Constants.cs]: This should be `"Microsoft.VisualStudio.Web.CodeGenerators.Mvc"`

**Comment by prafullbhosale** [comment]
The razor template files need to be updated
`@inherits Microsoft.Extensions.CodeGeneration.Templating.RazorTemplateBase`
                        :arrow_down: 
`@inherits Microsoft.VisualStudio.Web.CodeGeneration.Templating.RazorTemplateBase`

**Comment by prafullbhosale** [comment]
https://github.com/aspnet/Scaffolding/blob/release/src/Microsoft.Extensions.CodeGenerators.Mvc/Templates/ControllerGenerator/MvcControllerWithContext.cshtml#L251

Need to change `Microsoft.Extensions.CodeGeneration.Templating.TypeUtilities.IsTypePrimitive(primaryKeyType)`
:arrow_down: 
`Microsoft.VisualStudio.Web.CodeGeneration.Templating.TypeUtilities.IsTypePrimitive(primaryKeyType)`

**Comment by pranavkm** [comment]
Updated

**Comment by prafullbhosale** [comment]
:shipit:

**Pull request closed without merging**